src/core/osmLayer.js

```diff
@@ -55,6 +59,10 @@ geo.osmLayer = function (arg) {
     m_imageFormat = arg.imageFormat;
   }
 
+  if (arg && arg.displayLast !== undefined) {
```

Comment:
Yeah, I think that would be the reasonable expectation.  If it isn't there, then default to true, otherwise use it as a boolean.